app/jobs/send_webhook_job.rb

```diff
@@ -15,6 +15,7 @@ class SendWebhookJob < ApplicationJob
     'invoice.generated' => Webhooks::Invoices::GeneratedService,
     'invoice.drafted' => Webhooks::Invoices::DraftedService,
     'invoice.voided' => Webhooks::Invoices::VoidedService,
+    'invoice.payment_dispute_lost' => Webhooks::Invoices::PaymentDisputeLostService,
```

Comment:
We also have a new attribute to add to open api and the docs so maybe we can join forces in the update 🙌